quark/Evaluator/pyeval.py

```diff
@@ -1,32 +1,65 @@
+# Thanks for the description of Dalvik bytecode instruction from the following websites, some of our explanations or
+# comments will quote from it.
+# https://source.android.google.cn/devices/tech/dalvik/instruction-formats
+# http://pallergabor.uw.hu/androidblog/dalvik_opcodes.html
+
+import logging
+from datetime import datetime
+
 from quark.Objects.TableObject import TableObject
 from quark.Objects.VarabileObject import VarabileObject
 
 MAX_REG_COUNT = 40
+TIMESTAMPS = datetime.now().strftime('%Y-%m-%d-%H-%M')
+LOG_FILENAME = f"{TIMESTAMPS}.quark.log"
```

Comment:
Is it necessary to rotate the log by minutes?

Reply:
It will only be executed once, for example ("2020-01-08-13-29.quark.log") to log which command has executed when the quark instruction is executed, or maybe I misunderstand this usage?

Reply:
Apparently, it will create a new log file every execution. Since our engine is command line tool, not a running service. It is OK to separate each runtime log if this is what we want. How do you think ?

Reply:
I see. Any ideas on how to modify it?

Reply:
Actually, I prefer to rotate the log file by day, not by minute.

Reply:
got it!, I will change it by day!